src/Mockingbird/Utils/Default.swift

```diff
@@ -24,10 +25,51 @@ final class Default {
     public enum Folder {
 
         static var main = "/Users/" + NSUserName() + "/.mockingbird"
-        static var capture: String { main + "/capture" }
-        static var data: String { main + "/data" }
         static var mitm: String { main + "/mitmproxy" }
-        static var record: String { main + "/record" }
-        static var test: String { main + "/test" }
+        static var capture: String { workingDirectory + "/capture" }
+        static var record: String { workingDirectory + "/record" }
+        static var data: String { workingDirectory + "/data" }
+        static var test: String { workingDirectory + "/test" }
+
+        static var workingDirectory = ""
+
+        static var savedWorkingDirectory: String {
+
+            get {
+
+                guard let file = try? File(path: "\(Default.Folder.main)/configurations.json"),
```

Comment:
should we use codable here?

Reply:
Used codable after retrieving the data from the file. Take a new look!